src/freenet/node/useralerts/UserAlert.java

```diff
@@ -50,6 +60,18 @@ public interface UserAlert {
 	
 	public String dismissButtonText();
 	
+	public default String replyButtonText() {
```

Comment:
The newly added methods feel out of place on `UserAlert`, as they are not so much about the alert itself, but quite specific to N2N messaging.

If we would replace the `if (userAlert.canReply()) {` in `UserAlertManager` to check if the UserAlert is an `N2NTMUserAlert`, we don't need to pollute this interface with N2N messaging specifics.